ui/ducks/metamask/metamask.js

```diff
@@ -44,6 +44,7 @@ const initialState = {
     showFiatInTestnets: false,
     showTestNetworks: false,
     useNativeCurrencyAsPrimaryCurrency: true,
+    petnamesEnabled: true,
```

Comment:
This is a big decision we'll want to verify with Barbara.

My instinct is to disable it by default but include a whats new popup to highlight it for users.

Reply:
Matt and Derek discussed with Barbara: toggle will be enabled by default.

Still TODO: How to make users aware of new petnames feature.

Reply:
For the sake of modularity and separate PRs, should we merge this as is and introduce the whats new changes in a separate PR prior to the next release cut?

Reply:
We have the go-ahead to merge this now.